File: MODUL-5/SOAL 2/PRAK502-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>
#include <math.h>

int hitung (int nilai1, int nilai2) {
    return sqrt((nilai2 - nilai1) * (nilai2 - nilai1));
}

int mutlak(int angka) {
    if (angka < 0) {
        return -angka;
    }
    return angka;
}

int main () {
    int a, b, c, d, Hasil;

    scanf("%d", &a);
    scanf("%d", &c);
    scanf("%d", &b);
    scanf("%d", &d);

    Hasil = hitung(a, b) + hitung(c, d);
    printf("%d", mutlak(Hasil));

    return 0;
}

File: MODUL-5/SOAL 2/PRAK502-2410817120017-RIKAFAULIANARAHMI.py
import math

def hitung (nilai1, nilai2):
    return math.sqrt((nilai2 - nilai1) * (nilai2 - nilai1))

def mutlak(angka):
    if angka < 0:
        return -angka
    return angka

a, c, b, d = map(int, input().split())

hasil = hitung(a, b) + hitung(c, d)

print(f"{mutlak(hasil):.0f}")
